sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/crypto/_models.py

```diff
@@ -255,7 +274,14 @@ def recover_data_from_signature(
             :class:`~cryptography.exceptions.InvalidSignature` if the signature is invalid.
             :class:`~cryptography.exceptions.UnsupportedAlgorithm` if the signature data recovery is not supported with
             the provided `padding` type.
+            ValueError if the client is unable to obtain the key material from Key Vault.
```

Comment:
This formatting doesn't render so well - I would break it up into multiple `:raises:` statements, one for each of the error types:
<img width="489" alt="image" src="https://github.com/Azure/azure-sdk-for-python/assets/8689453/63032e2d-e906-4a57-a4ae-528de614c035">

Reply:
Updated, thank you for the suggestion!